data/geany.glade

```diff
@@ -7228,7 +7228,7 @@
                                   <object class="GtkMenuItem" id="goto_tag_declaration1">
                                     <property name="visible">True</property>
                                     <property name="can_focus">False</property>
-                                    <property name="label" translatable="yes">Go to T_ag Declaration</property>
+                                    <property name="label" translatable="yes">Go to Symbol Dec_laration</property>
```

Comment:
The mnemonic changes, any reason?  IMO it's not a great idea, esp. as I don't find the new particularly great